tests/test_unittest/test_gtc/test_naive_python_gen.py

```diff
@@ -80,99 +81,73 @@ def test_binary_op(naive_codegen, binary_operator):
-
-
-def test_vertical_interval(naive_codegen):
-    vertical_interval = VerticalInterval(
-        start=AxisBound.start(),
-        end=AxisBound.end(),
-        horizontal_loops=[
-            HorizontalLoop(
-                stmt=AssignStmt(
-                    left=FieldAccess.centered(name="a"), right=FieldAccess.centered(name="b")
-                )
-            ),
-            HorizontalLoop(
-                stmt=AssignStmt(
-                    left=FieldAccess.centered(name="b"), right=FieldAccess.centered(name="c")
-                )
-            ),
-        ],
-    )
-    source_tree = ast_parse(naive_codegen.apply(vertical_interval))
-    for_k = source_tree.body[0]
-    assert isinstance(for_k, ast.For)
-    assert for_k.target.id == "K"
-    assert isinstance(for_k.iter, ast.Call)
-    assert isinstance(for_k.iter.args[-1], ast.Subscript)
-    for_i = for_k.body[0]
-    assert isinstance(for_i, ast.For)
-    assert for_i.target.id == "I"
-    for_i = for_k.body[1]
-    assert isinstance(for_i, ast.For)
-    assert for_i.target.id == "I"
-
-
-def test_vertical_loop(naive_codegen, loop_order):
-    vertical_loop = VerticalLoop(
-        loop_order=loop_order,
-        vertical_intervals=[
-            VerticalInterval(
-                start=AxisBound.start(),
-                end=AxisBound.from_end(2),
-                horizontal_loops=[
-                    HorizontalLoop(
-                        stmt=AssignStmt(
-                            left=FieldAccess.centered(name="a"),
-                            right=FieldAccess.centered(name="b"),
-                        )
-                    )
-                ],
-            ),
-            VerticalInterval(
-                start=AxisBound.from_end(1),
-                end=AxisBound.end(),
-                horizontal_loops=[
-                    HorizontalLoop(
-                        stmt=AssignStmt(
-                            left=FieldAccess.centered(name="a"),
-                            right=FieldAccess.centered(name="b"),
-                        )
-                    )
-                ],
-            ),
-        ],
-    )
-    source_tree = ast_parse(naive_codegen.apply(vertical_loop))
-
-    for_k = source_tree.body[0]
-    assert isinstance(for_k, ast.For)
-    assert for_k.target.id == "K"
-    assert isinstance(for_k.iter, ast.Call)
-    range_end = for_k.iter.args[-1]
-    assert isinstance(range_end, ast.BinOp)
-    assert isinstance(range_end.op, ast.Sub)
-    assert range_end.right.n == 2
-
-    for_k = source_tree.body[1]
-    assert isinstance(for_k, ast.For)
-    assert for_k.target.id == "K"
-    assert isinstance(for_k.iter, ast.Call)
-    range_start = for_k.iter.args[0]
-    range_end = for_k.iter.args[-1]
-    assert isinstance(range_start, ast.BinOp)
-    assert isinstance(range_end, ast.Subscript)
-    assert isinstance(range_start.op, ast.Sub)
-    assert range_start.right.n == 1
+# def test_vertical_interval(naive_codegen):
```

Comment:
TODO discuss with @egparedes if we can improve errors in TemplateGenerator

Reply:
this is a jinja feature. Should we try to set the default to strict (if possible) or provide an option?